src/platforms/android/configuration/integrations/okhttp.mdx

```diff
@@ -239,3 +239,102 @@ private final OkHttpClient client = new OkHttpClient.Builder()
   .addInterceptor(new SentryOkHttpInterceptor(new CustomBeforeSpanCallback()))
   .build();
 ```
+
+## HTTP Client Errors
+
+HTTP client errors, like bad response code, are captured as error events and reported to Sentry. The error event will contain the `request` and `response` data, such as `url`, `status_code`, and so on.
```

Comment:
`m`: I think we could be more explicit to explain that this feature captures these errors automatically.
```suggestion
This feature, once enabled, automatically captures HTTP client errors, like bad response codes, as error events and reports them to Sentry. The error event will contain the `request` and `response` data, such as `url`, `status_code`, and so on.
```